docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -3553,3 +3553,17 @@ function sitenow_update_9022() {
 function sitenow_update_9023() {
   sitenow_update_9020();
 }
+
+/**
+ * Remove headline content from footer block before setting perms.
+ */
+function sitenow_update_9024() {
+  /** @var \Drupal\Core\Plugin\DefaultPluginManager $filters */
+  $filters = \Drupal::service('plugin.manager.config_filter')->getDefinitions();
```

Comment:
Is $filters an unused variable? I don't see it called in the code below this

Reply:
RIGHT! I was going to call all of the config splits to make sure it was fixed on every site then realized this is a universal issue on our broad platform.